frameworks/g-cloud-9/questions/services/workingWithMobile.yml

```diff
@@ -0,0 +1,17 @@
+name: Working with mobile
+question: Has your service been designed to work with mobile?
+depends:
+  - "on": "lot"
+    being:
+      - cloud-hosting
+      - cloud-software
+      - cloud-support
+
+type: boolean
+
+validations:
+   -
+     name: answer_required
+     message: 'You need to answer this question.'
+
+optional: false
```

Comment:
Don't need `optional: false` here - the existence of an `answer_required` validation should enforce that for you.
I see it's on a few of the multiquestions, and maybe it does something useful there, but definitely not needed for a boolean as far as I can think.